website/docs/r/iothub.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_iothub"
+sidebar_current: "docs-azurerm-resource-iothub"
+description: |-
+  Creates a new IotHub resource 
+---
+
+# azurerm\_iothub
+
+Creates a new IotHub
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "test" {
+  name     = "resourceGroup1"
+  location = "West US"
+}
+
+
+resource "azurerm_iothub" "test" {
+	name                             = "test"
+	resource_group_name              = "${azurerm_resource_group.test.name}"
+	location                         = "westus"
+	sku {
+		name = "S1"
+		tier = "Standard"
+		capacity = "1"    
+	}
+
+	tags {
+		"purpose" = "testing"
+	    }
+	}
+
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the IotHub resource. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group under which the IotHub resource has to be created. Changing this forces a new resource to be created.
+
+* `location` - (Required) Specifies the supported Azure location where the resource has to be createc. Changing this forces a new resource to be created.
+
+* `sku` - (Required) Specifies the capacity and tier of the IotHub. 
+
+* `tags` - (Optional) A mapping of tags to assign to the resource.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - The IotHub ID.
+
+* `etag` - The etag of IotHub Resource.
+
+* `hostname` - The hostname of the IotHub Resource.
+
+* `shared_access_policy` - The list of access policy which contain access keys and permissions.
+
+
```

Comment:
can we add a section here with the information needed for Import, e.g.

"
## Import

IoTHubs can be imported using the `resource id`, e.g.

```shell
terraform import azurerm_iothub.hub1 /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mygroup1/providers/Microsoft.Devices/IotHubs/hub1
```
"